src/exceptions/GPL-CC-1.0.xml

```diff
@@ -0,0 +1,101 @@
+    </crossRefs>
+    <notes>
+      Single exception covering three variants:
+      Individual, Company, and Project.
+    </notes>
+    <titleText>
+      <optional>
+      <p>
+        GPL Cooperation Commitment Version 1.0
+      </p>
+      </optional>
+    </titleText>
+    <p>
+      <optional>Solely for any software for which I personally own
+      copyright that is licensed under a Covered License,</optional>
+      <alt name="before" match="before|Before">before</alt>
+      filing or continuing to prosecute any legal proceeding or claim
+      (other than a Defensive Action) arising from termination of a
+      Covered License,<alt name="subject" match="I|we|.*">we</alt>
+      commit<optional>s</optional>
+      to extend to the person or entity ("you") accused of violating
+      the Covered License the following provisions regarding cure and
+      reinstatement, taken from GPL version 3. As used here, the term
+      "this License" refers to the specific Covered License being enforced.
+    </p>
+    <p>
+      <optional>"</optional>However,
+      if you cease all violation of this License, then your
+      license from a particular copyright holder is reinstated (a)
+      provisionally, unless and until the copyright holder explicitly
+      and finally terminates your license, and (b) permanently, if
+      the copyright holder fails to notify you of the violation by
+      some reasonable means prior to 60 days after the cessation.
+    </p>
+    <p>
+      Moreover, your license from a particular copyright holder is
+      reinstated permanently if the copyright holder notifies you of
+      the violation by some reasonable means, this is the first time you
+      have received notice of violation of this License (for any work)
+      from that copyright holder, and you cure the violation prior to
+      30 days after your receipt of the notice.<optional>"</optional>
+    </p>
+    <p>
+      <alt name="subject" match="I|We|.*">We</alt>
+      intend<optional>s</optional>
+      this Commitment to be irrevocable, and binding and enforceable
+      against <alt name="object" match="me|us|.*">us</alt>
+      and assignees of or successors to
+      <alt name="possessive" match="my|our|.*">our</alt>
+      copyrights.
+    </p>
+    <optional>
+    <p>
+      <alt name="subject" match=".*">COMPANY</alt>
+      may modify this Commitment by publishing a new
+      edition on this page or a successor location.
+    </p>
+    </optional>
+    <p>
+      Definitions<optional>:</optional>
+    </p>
+    <p>
+      "Covered License" means the GNU General Public License, version
+      2 (GPLv2), the GNU Lesser General Public License, version 2.1
+      (LGPLv2.1), or the GNU Library General Public License, version
+      2 (LGPLv2), all as published by the Free Software Foundation.
+    </p>
+    <p>
+      "Defensive Action" means a legal proceeding or claim
+      that<alt name="subject" match="I|We|.*">We</alt>
```

Comment:
Same comment as above - Since .* will match any of the words, the other alternatives will be ignored.  I'm thinking we just restrict it to I and We and remove the .*

Reply:
same reply. need COMPANY_NAME